src/main/java/no/unit/services/PnxServices.java

```diff
@@ -138,6 +147,15 @@ protected JsonObject extractUsefulDataFromPnxService(JsonObject response) {
         return extractedData;
     }
 
+    protected String getMaskedPrimoRestApiKey(String apiKey){
+        int TYPICAL_API_KEY_MIN_LENGTH = 32;
+        if (apiKey.length() > TYPICAL_API_KEY_MIN_LENGTH) {
+            return "XXXXXX" + apiKey.substring(apiKey.length() - 4);
+        }else {
+            return "<API_KEY>";
+        }
+    }
+
```

Comment:
Skulle denne funksjonen vært flyttet til en egen fil? Jeg tenker den ikke hører hjemme her, men den hører heller ikke hjemme i Config, så da blir spørsmålet om den skulle vært i en egen klasse

Reply:
Har forøvrig ikke satt antall X'er til å matche antall bokstaver i api-nøkkelen. Hva synes du? skal jeg sette på én "X" per maskert character?

Reply:
Jeg synes det er greit at den ligger her 👍

Reply:
Helt greit at den ligger der. Men jeg ser at vi har jo ApiKey fra Config. Så en enkelt string replace ville har gjort jobben (måtte ikke gå via Pattern strengt tatt. Antal XXXXXer er ikke viktig.

Reply:
Fjernet pattern.